repartitions/models.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
```

Comment:
_Please_ add docstrings to explain what are these models.
What is the difference between a `Campaign`, a `UserCampaign`? Between a `Category` or a `Group` or a `Proposition`? There are six models here and their relationship is not linear, it is very hard to understand.